heartbeat/pgsql

```diff
@@ -1479,7 +1479,7 @@ set_async_mode_all() {
 }
 
 set_async_mode() {
-    cat $REP_MODE_CONF |  grep -q -e "[,' ]$1[,' ]"
+    cat $REP_MODE_CONF |  grep -q -e "\"$1\""
```

Comment:
I think that the backward compatibility is kept with` \"$1\"`.
The pattern `[,' ]$1[,' ]` is maybe to ensure uniqueness of $1. 

ex) if $1 is test, test matches both.
```
synchronous_standby_names = 'test, testtest'
```
Now, pgsql RA sets " " to all elements of synchronous_standby_names regardless of PostgreSQL version, and the pattern `\"$1\"` satisfies uniqueness.
```
synchronous_standby_names = '"test", "testtest"'
```